README.md

```diff
@@ -21,6 +21,12 @@ Inside the project, install the project's dependencies:
 cd igbo_api/
 yarn install
 ```
+If you are running the API on Windows operating system, run the module to install it globally:
+
+```
+cd igbo_api/                   
+yarn install --save-optional win-node-env
```

Comment:
The correct command is `yarn add --optional win-node-env`

Reply:
okay on it